audit/format.go

```diff
@@ -127,6 +128,11 @@ func (f *AuditFormatter) FormatRequest(ctx context.Context, w io.Writer, config
 		},
 	}
 
+	var zt time.Time
+	if auth.IssueTime != zt {
```

Comment:
I used to do this all the time, but I think you can now use the `.IsZero` method instead.

Reply:
ah, much better.  Was not liking that construct.